hrm-service/src/case/case-routes-v0.ts

```diff
@@ -8,6 +8,18 @@ import { getCase } from './case';
 
 const casesRouter = SafeRouter();
 
+/**
+ * Returns a filterable list of cases for a helpline
+ *
+ * @param {string} req.accountSid - SID of the helpline
+ * @param {CaseListConfiguration.sortDirection} req.query.sortDirection - Sort direction
+ * @param {CaseListConfiguration.sortBy} req.query.sortBy - Sort by
+ * @param {CaseListConfiguration.limit} req.query.limit - Limit
+ * @param {CaseListConfiguration.offset} req.query.offset - Offset
+ * @param {import('./case').SearchParameters} req.query.search
+ *
+ * @returns {import('./case').CaseSearchReturn} - List of cases
+ */
```

Comment:
I'm not 100% convinced about having this jsdocs annotations, as they might confuse more than help (those aren't the actual types of the `req` but what we want them to be). I don't think they'll hurt me anyways. Deferring to someone's else opinion if they have a stronger one.

Reply:
Could we mitigate the need for calling out types in the JSDoc by explicitly setting them in the route signature. Then the docs could purely focus on the semantic meaning of the parameters rather than having to worry about describing the types